src/elements/kw-social-comment/kw-social-comment.html

```diff
@@ -126,15 +135,16 @@
             }
             
             span.bold {
-                font-weight: bold
+                font-weight: bold;
+                margin-right: 5px;
             }
         </style>
         <div class="comments-box">
-            <h3 hidden$="{{comments.length > 0}}">Be the first to comment!</h3>
+            <h3 hidden$="{{_computeHidden(comments)}}">Be the first to comment!</h3>
             <div class="line"></div>
             <template is="dom-repeat" items="{{comments}}">
                 <div class="comment">
-                    <div class="comment-avatar"><img src="{{item.author.avatar.urls.circle}}" /></div>
+                    <div class="comment-avatar"><img src="{{_getAvatar(item)}}" /></div>
```

Comment:
As @vikdoro said somewhere else, using an iron image would allow you to have a placeholder and a preload/fade animation when the browser finished loading